lib/data_types/tracking_state.dart

```diff
@@ -0,0 +1,26 @@
+enum TrackingStateChange {
+  /// Tracking is active
+  start,
+
+  /// Tracking stopped
+  stop,
+
+  /// Tracking doesn't happen due to missing permission(s)
+  permissionsDenied,
+
+  /// Tracking doesn't happend due to location service being disabled.
+  locationDisabled,
+
+  ///  SDK provided with invalid token.
+  invalidPublishableKey,
+
+  /// SDK encountered network error
+  networkError,
+
+  /// Tracking won't happen as either account is suspended or device was deleted
+  /// from account.
+  authError,
+
+  /// SDK encountered a unknown issue, will log stack in this case
+  unknownError
```

Comment:
i'd put errors into a separate publisher and not into the tracking state change

Reply:
yes, it will be this way is further updates